regnety/dataset/imagenet.py

```diff
@@ -0,0 +1,223 @@
+import tensorflow as tf
+import os
+
+from dataclasses import dataclass
+from official.vision.image_classification.augment import RandAugment
+from typing import Union, Callable, Tuple, List, Type
+
+_TFRECS_FORMAT = {
+        "image": tf.io.FixedLenFeature([], tf.string),
+        "height": tf.io.FixedLenFeature([], tf.int64),
+        "width": tf.io.FixedLenFeature([], tf.int64),
+        "filename": tf.io.FixedLenFeature([], tf.string),
+        "label": tf.io.FixedLenFeature([], tf.int64),
+        "synset": tf.io.FixedLenFeature([], tf.string),
+}
+    
+
+class ImageNet:
+    """Class for all ImageNet related functions, includes TFRecords loading,
+    preprocessing and augmentations. TFRecords must follow the format given
```

Comment:
How about changing the first sentence to the following?

```
Class for all ImageNet data-related functions, including TFRecord parsing along with augmentation transforms. 
```

Where have you specified the TFRecord format?

Reply:
Sure, I'll change that